src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -1552,9 +1768,9 @@ static void Main()
                 // (9,9): error CS0411: The type arguments for method 'Program.AsArray<T>(T[])' cannot be inferred from the usage. Try specifying the type arguments explicitly.
                 //         AsArray([]);
                 Diagnostic(ErrorCode.ERR_CantInferMethTypeArgs, "AsArray").WithArguments("Program.AsArray<T>(T[])").WithLocation(9, 9),
-                // (10,21): error CS0037: Cannot convert null to 'int' because it is a non-nullable value type
+                // (10,17): error CS1503: Argument 1: cannot convert from 'collection expressions' to 'int[]'
```

Comment:
This feels like a less than ideal diagnostic experience. I don't think there's anything to do about it in this PR, though.